Analysis/src/QwOptions.cc

```diff
@@ -397,3 +397,16 @@ std::pair<int,int> QwOptions::GetIntValuePair(const std::string& key)
 
   return mypair;
 }
+
+void QwRootFile::DefineOptions(QwOptions &options)
```

Comment:
There is already a `DefineOptions` method defined earlier in `QwOptions` and also in `QwRootFile` class which is in `QwRootFile.cc`. This is a redefinition. So the body of this method should be moved to body of `DefineOptions` in `QwRootFile` class.